code/modules/mob/living/living_defines.dm

```diff
@@ -110,6 +110,7 @@
 	var/list/obj/effect/proc_holder/abilities = list()
 
 	var/can_be_held = FALSE	//whether this can be picked up and held.
+	var/can_head = FALSE //if it can be held, can it fit on peoples heads?

```

Comment:
We're not short on letters, you could name this something specific.

Also probably better off to just make this generic designated slots instead. (carrying monkeys/sloths on your back or whatever)